Documentation/contributing/testing/workflows.rst

```diff
@@ -0,0 +1,118 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _workflows:
+
+GitHub Workflows Testing Framework
+==================================
+
+Running GitHub Workflows Locally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Running GitHub workflows locally is currently possible with the help of `nektos/act <https://github.com/nektos/act>`_.
+
+Requirements:
+ - ``act`` - with commit ``bf4aad6ad296822cb0352e21ce3867284eaf1eff``, build locally by cloning the repository and running ``make build && alias act=$PWD/dist/local/act``
+ - ``docker``
+
+Currently, only the following workflows were tested to work with ``act``:
+
+ - .github/workflows/conformance-gke.yaml - :ref:`conformance_gke`
+
+.. _conformance_gke:
+
+Running Conformance GKE
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Workflow Requirements:
+ - :ref:`gh_token`
+ - :ref:`gcp_project_id`
+ - :ref:`gcp_sa_key`
+
+
+Once you have ``GITHUB_TOKEN`` exported and ``secrets-file.txt`` created, run
+the following command. As you can see, ``git rev-parse HEAD`` will be used as
+the default commit but you can choose any commit that has a docker image
+available:
+
+::
+
+    act -s GITHUB_TOKEN=${GITHUB_TOKEN} \
+        --secret-file=secrets-file.txt \
+        --env ACT_SHA=`git rev-parse HEAD` \
```

Comment:
Nit: Should we recommend `$(...)` syntax instead? Shellcheck complains about the ` synxtax

Reply:
Here and for the `SECRET=` commands